go/service/kvstore.go

```diff
@@ -44,21 +44,34 @@ func (h *KVStoreHandler) assertLoggedIn(ctx context.Context) error {
 	return nil
 }
 
+func rewriteMisleadingError(err error) error {
+	if err == nil {
+		return nil
+	}
+	return libkb.AppStatusError{
+		Code: libkb.SCTeamReadError,
+		Desc: "You are not a member of this team",
+	}
+}
+
 func (h *KVStoreHandler) resolveTeam(mctx libkb.MetaContext, userInputTeamName string) (teamID keybase1.TeamID, err error) {
+	if strings.Contains(userInputTeamName, "@") {
```

Comment:
why isn't SBS supported?

Reply:
i assume we'd have to re-encrypt all the entries. i didn't actually check this. do you think it would probably work out of the box?

Reply:
People with access to the application or team key at generation n also have access to all previous generations <=n. So for SBS, that'd be once you're keyed in as a cryptomember. So yes, I'd expect it to work out of the box.

Rekeying entries so that old devices don't have access is another thing. The opposite of useability. And that ought to be the same regardless of SBS.